packages/xeus-core/src/web.worker.kernel.base.ts

```diff
@@ -287,6 +208,31 @@ export class WebWorkerKernel implements IKernel {
     return this._disposed;
   }
 
+  /**
+   * Get the content manager
+   */
+  protected get contentsManager() {
+    return this._contentsManager;
+  }
+
+  /**
+   * Get the content processor
+   */
+  protected get contentsProcessor() {
+    return this._contentsProcessor;
+  }
+
+  /**
+   * Get the worker
+   */
+  protected get worker() {
+    return this._worker;
+  }
+
+  protected get remoteKernel() {
+    return this._remoteKernel;
+  }
```

Comment:
Marking these getter protected while the real attributes are private is not really helpful. We could either drop the protected flag or make all attributes protected

Reply:
indeed 👍🏽